central/convert/storagetov2/compliance_profile_service.go

```diff
@@ -73,11 +73,6 @@ func ComplianceProfileSummary(incoming []*storage.ComplianceOperatorProfileV2, c
 
 	summaries := make([]*v2.ComplianceProfileSummary, 0, len(profileSummaryMap))
 	for _, profileName := range orderedProfiles {
-		// Verify the profile is in all clusters
```

Comment:
This is now taken care of by the calls from the service to the datastore, so by the time we get here we have the data we want.